public/app/features/explore/TimeSyncButton.test.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { TimeSyncButton } from './TimeSyncButton';
+import { mount } from 'enzyme';
+
+const setup = (isSynced: boolean) => {
+  const onClick = () => {};
+  return mount(<TimeSyncButton onClick={onClick} isSynced={isSynced} />);
+};
+
+describe('TimeSyncButton', () => {
+  it('should render component', () => {
+    const wrapper = setup(true);
+    expect(wrapper).toMatchSnapshot();
+  });
+  it('should change style when synced', () => {
+    const wrapper = setup(true);
+    expect(wrapper.find('button').hasClass('css-14r9fpj-timePickerSynced')).toEqual(true);
```

Comment:
Instead of using class nam(any change to css will make this test to fail), can we use `aria-label` instead?

Reply:
Absolutely! Changed. :)